synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -943,37 +943,65 @@ CanvasInterface::import(
 
 bool
 CanvasInterface::import_sequence(
-		const std::set<synfig::String> &filenames,
-		synfig::String &errors,
-		synfig::String &/*warnings*/,
-		bool resize_image,
-		UIInterface::Response rem_dups)
+        const std::set<synfig::String> &filenames,
+        synfig::String &errors,
+        synfig::String &/*warnings*/,
+        bool resize_image,
+        int rem_dups)
```

Comment:
I really think it should be a simple bool :)
the caller could be:
`import_sequence(filenames, errors, warnings, App::resize_imported_images,answer == UIInterface::RESPONSE_YES);`

Reply:
Yes Changed it.. :sweat_smile: